lib/kennel/tasks.rb

```diff
@@ -71,18 +109,7 @@ def abort(message = nil)
 
   desc "update on push to the default branch, otherwise show plan"
   task :ci do
-    branch = (ENV["TRAVIS_BRANCH"] || ENV["GITHUB_REF"]).to_s.sub(/^refs\/heads\//, "")
-    on_default_branch = (branch == (ENV["DEFAULT_BRANCH"] || "master"))
-    is_push = (ENV["TRAVIS_PULL_REQUEST"] == "false" || ENV["GITHUB_EVENT_NAME"] == "push")
-    task_name =
-      if on_default_branch && is_push
-        Kennel.strict_imports = false
-        "kennel:update_datadog"
-      else
-        "kennel:plan" # show plan in CI logs
-      end
-
-    Rake::Task[task_name].invoke
+    Kennel::Tasks.ci
```

Comment:
Come to think of it, I'm not sure that `:ci` needed to move out of rake and into a normal method. But it does no harm.

Reply:
would rather keep it then, to reduce indirection, but either way is fine

Reply:
I'd rather keep it where it is – class/module code is easier to reuse / refactor / test. Rake task code should ideally be thin layer over the top of that.